### PR TITLE
db-dj-docker demo

### PR DESCRIPTION
**Pull request by ChunhuiY**

### CODE REVIEW

**Comment by ChunhuiY** [comment]
There are two problems that l tried to fix many times but they are still there.
Terminal keeps showing: docker-compose.yml: version is obsoleteThe default page and connection were launched successfully.
l tried creating some simple views and configuring the URls to buildthe project further, but they are not being displayed.

**Review by LeonQC — COMMENTED** [review]
Please remove all the temporary folder/files such as __pycache__

File: dj-docker-db demo/djangoProject3/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Dockerfile
+
+# 使用官方的Python基础镜像
+FROM python:3.8-slim
```
Comment (LeonQC) [inline dj-docker-db demo/djangoProject3/Dockerfile]: Try to use 3.11+

**Review by LeonQC — APPROVED** [review]

**Comment by LeonQC** [comment]
need resolve the conflict, otherwise you  won't be able to merge

**Pull request closed without merging**